backerymails.post_update.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Backerymails module.
+ */
+
+use Drupal\Core\Database\Database;
+
+/**
+ * Migrate data from the old existing Backerymails table to the new one.
+ */
+function backerymails_post_update_migrate_data(&$sandbox = NULL) {
```

Comment:
from the [documentation](https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Extension!module.api.php/function/hook_post_update_NAME/):

> NAME can be arbitrary machine names. In contrast to hook_update_N() the alphanumeric naming of functions in the file is the only thing which ensures the execution order of those functions. If update order is mandatory, you should add numerical prefix to NAME or make it completely numerical.
> 
> Drupal also ensures to not execute the same hook_post_update_NAME() function twice.

So, let's prefix this with a number `backerymails_post_update_8001_migrate_data` ?

```suggestion
function backerymails_post_update_8001_migrate_data(&$sandbox = NULL) {
```

Reply:
right